bower.json

```diff
@@ -20,19 +20,19 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-arrays": "^4.0.0",
-    "purescript-either": "^3.0.0",
-    "purescript-foldable-traversable": "^3.0.0",
-    "purescript-identity": "^3.0.0",
-    "purescript-integers": "^3.0.0",
-    "purescript-lists": "^4.0.0",
-    "purescript-maybe": "^3.0.0",
-    "purescript-strings": "^3.0.0",
-    "purescript-transformers": "^3.0.0",
-    "purescript-unicode": "^3.0.0"
+    "purescript-arrays": "#compiler/0.12",
+    "purescript-either": "#compiler/0.12",
+    "purescript-foldable-traversable": "#compiler/0.12",
+    "purescript-identity": "#compiler/0.12",
+    "purescript-integers": "#compiler/0.12",
+    "purescript-lists": "#compiler/0.12",
+    "purescript-maybe": "#compiler/0.12",
+    "purescript-strings": "#compiler/0.12",
+    "purescript-transformers": "#compiler/0.12",
+    "purescript-unicode": "justinwoo/purescript-unicode#compiler/0.12"
```

Comment:
should be changed over after https://github.com/purescript-contrib/purescript-unicode/pull/16